integration-tests/solr/src/test/java/org/apache/camel/quarkus/component/solr/it/SolrTestResource.java

```diff
@@ -111,10 +112,17 @@ private void createSslContainer() {
      * creates a cloud container with zookeeper
      */
     private void createCloudContainer() {
-        cloudContainer = new DockerComposeContainer(new File("src/test/resources/cloud-docker-compose.yml"))
-                .withExposedService("solr1", SOLR_PORT)
-                .withExposedService("zoo1", ZOOKEEPER_PORT)
-                .waitingFor("create-collection", Wait.forLogMessage(".*Created collection 'collection1'.*", 1));
+        if (SystemUtils.IS_OS_LINUX) {
+            cloudContainer = new DockerComposeContainer(new File("src/test/resources/cloud-docker-compose.yml"))
```

Comment:
Can these be read from the classpath? I'm not sure resolving from the source tree will work on the `quarkus-platform` build.

Reply:
@jamesnetherton , pushed a new commit, please take a look.
Thanks!
Freeman